docs/jwst/stpipe/config_asdf.rst

```diff
@@ -27,43 +33,51 @@ The remaining sections will describe the file format and contents.
 File Contents
 -------------
 
-To describe the contents of an ASDF file, the configuration for the step ``CubeBuildStep`` will be used as the example:
+To describe the contents of an ASDF file, the configuration for the step
+``CubeBuildStep`` will be used as the example:
 
 .. code-block::
 
-    #ASDF 1.0.0
-    #ASDF_STANDARD 1.3.0
-    %YAML 1.1
-    %TAG ! tag:stsci.edu:asdf/
-    --- !core/asdf-1.1.0
-    asdf_library: !core/software-1.0.0 {author: Space Telescope Science Institute, homepage: 'http://github.com/spacetelescope/asdf',
-      name: asdf, version: 2.4.2}
-    history:
-      entries:
-      - !core/history_entry-1.0.0 {description: Base values, time: !!timestamp '2019-10-29
-          21:20:50'}
-      extensions:
-      - !core/extension_metadata-1.0.0
-        extension_class: asdf.extension.BuiltinExtension
-        software: {name: asdf, version: 2.4.2}
-    meta:
-      author: SPECIFY AUTHOR
-      date: '2019-10-29T16:02:59.377'
-      description: Parameters for calibration step SPECIFY
-      filename: pars-cubebuildstep.asdf
-      instrument: {name: SPECIFY}
-      model_type: StepParsModel
-      origin: STScI
-      pedigree: SPECIFY PEDIGREE
-      reftype: pars-cubebuildstep
-      telescope: JWST
-      useafter: SPECIFY
-    parameters: {band: all, channel: all, class: jwst.cube_build.cube_build_step.CubeBuildStep,
-      coord_system: world, filter: all, grating: all, name: CubeBuildStep, output_type: band,
-      output_use_model: true, rois: 0.0, roiw: 0.0, scale1: 0.0, scale2: 0.0, scalew: 0.0,
-      search_output_file: false, single: false, skip_dqflagging: false, weight_power: 2.0,
-      weighting: msm}
-    ...
+   #ASDF 1.0.0
+   #ASDF_STANDARD 1.5.0
+   %YAML 1.1
+   %TAG ! tag:stsci.edu:asdf/
+   --- !core/asdf-1.1.0
+   asdf_library: !core/software-1.0.0 {author: Space Telescope Science Institute, homepage: 'http://github.com/spacetelescope/asdf',
+     name: asdf, version: 2.7.3}
+   history:
+     extensions:
+     - !core/extension_metadata-1.0.0
+       extension_class: asdf.extension.BuiltinExtension
+       software: !core/software-1.0.0 {name: asdf, version: 2.7.3}
+   class: jwst.cube_build.cube_build_step.CubeBuildStep
+   name: CubeBuildStep
+   parameters:
+     band: all
+     channel: all
+     coord_system: skyalign
+     filter: all
+     grating: all
+     input_dir: ''
+     output_ext: .fits
+     output_type: band
+     output_use_index: true
+     output_use_model: true
+     post_hooks: []
+     pre_hooks: []
```

Comment:
None of these general Step parameters applicable to all steps should ever be included in a parameter reference file.  And in fact most of a step-specific parameters shouldn't either.  Generally, the rule should be that if you are selecting on an instrument with CRDS selectors, only those parameters that change from the code default based on instrument and CRDS selectors should be included.